code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -26,6 +26,16 @@
 		else
 			to_chat(src, "<span class ='notice'>You're already using that style.</span>")
 
+/mob/living/carbon/human/proc/change_pronoun()
```

Comment:
Not sure how that makes sense as a verb, if anything, should be a preference?

Reply:
It's already a preference. This is so it could be changed during the round, i.e. if you change your haircut and it would make your character look more masculine/feminine, etc. Also useful for trying to disguise as someone of the opposite sex as a traitor; they could still tell from your sprite but they'd have to do a little more work to figure it out.
It's also excluded from dionaea, vaurca, and non-shell IPCs for now.